modules/data/about_prefs.components.json

```diff
@@ -8,7 +8,6 @@
             "searchEngineDropdown"
         ]
     },
-
```

Comment:
Were these line deletions done automatically by PyCharm? I kind of prefer the blank lines for readability, but it's not a dealbreaker.

Reply:
On reformat file (Ctrl + Alt + L ) is the pycharm doing. I added the spaces again to match the rest of the json files